examples/pip_install/BUILD

```
@@ -1,4 +1,5 @@
 load("@pip//:requirements.bzl", "requirement")
+load("@rules_python//python/pip_install:pip_requirements.bzl", "pip_requirements")
 load("@rules_python//python:defs.bzl", "py_binary", "py_test")
```

```suggestion
load("@rules_python//python:defs.bzl", "py_binary", "py_test")
load("@rules_python//python/pip_install:pip_requirements.bzl", "pip_requirements")
```

will fix the `buildifier` error that's failing CI check.

---

thx